tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -48,6 +48,11 @@ public partial class SpanContext : ISpanContext, IReadOnlyDictionary<string, str
         /// </summary>
         public static readonly ISpanContext None = new ReadOnlySpanContext(traceId: Trace.TraceId.Zero, spanId: 0, serviceName: null);
 
+        /// <summary>
+        /// A SpanContext with all fields set to zero or empty values, for span pointers.
+        /// </summary>
+        public static readonly SpanContext ZeroContext = new();
```

Comment:
Instead of adding `ZeroContext`, I you can repurpose `None` in the line above, since it's no longer part of the public api. Try to combine both into something like

```csharp
public static readonly SpanContext None = new();
```

If you can't because it's used somewhere else, then at least rename this one to `Zero` to follow .NET naming conventions (like `TimeSpan.Zero` and `Decimal.Zero`).